pkgs/tools/misc/ethminer/default.nix

```
@@ -1,5 +1,6 @@
 {
   lib,
+  stdenv,
   clangStdenv,
   fetchFromGitHub,
```

```suggestion
{ lib
, stdenv
, clangStdenv
, fetchFromGitHub
```

Please format the inputs in this style.

---

Could we leave style nitpickings on portions I didn't even really touch out of this? Thanks.

If you want to "fix" code style (we don't even have a style guide) feel free to make PRs specific to that purpose (I'd genuinely like that!) but please don't ask me to do that for you in PRs which are not.  
I get it, style is important. I try to "fix" sections which I perceive as not conforming to the "norms" when there's a good opportunity (i.e. I'm reworking it in large parts anyways) but you cannot expect people to go out of their way to do that sort of thing where it wouldn't be necessary.

I didn't open ethminer's default.nix and said to myself "Boy golly, I'm going to fix this file's styling!", I opened it to make a functional change with actual value to users.

As a contributor, I see such reviews as entirely unnecessary and even discouraging.